etna/series/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from wagtail.admin.panels import FieldPanel
+from wagtail.search import index
+
+from wagtailmetadata.utils import get_image_model_string
+
+
+class SeriesCategory(models.TextChoices):
+    RECORD_COLLECTION = "Record collection", "Record collection"
+
+
+class Series(index.Indexed, models.Model):
```

Comment:
What does `index.Indexed` do? I've not seen this used before

Reply:
This enables the model to be searchable with the Wagtail Search.
https://docs.wagtail.org/en/stable/topics/search/index.html

Now thinking about it, I am not sure if the new Wagtail `SnippetViewSet` implementation uses this or uses the old-school Django admin search method where it just filters the database with `__icontains`.

Reply:
I've just looked at other TNA snippets in the CMS and the search for other snippets doesn't seem to show up at all. The lack of `index.Indexed` is likely a culprit.

Reply:
Cool, thanks Tomasz - good to know! 😄